Sources/TokamakLint/Reporters/TokamakLogger.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  TokamakLogger.swift
+//  TokamakLint
+//
+//  Created by Matvii Hodovaniuk on 5/30/19.
+//
+
+import Foundation
+import Logging
+
+enum Output {
+  case stdout
+  case file
+}
+
+public struct Outputs: OptionSet {
+  public init(rawValue: Outputs.RawValue) {
+    self.rawValue = rawValue
+  }
+
+  public let rawValue: Int
+
+  static let stdout = Outputs(rawValue: 1)
+  static let file = Outputs(rawValue: 2)
+}
+
+public struct TokamakLogger: LogHandler {
+  private static var overrideLogLevel: Logger.Level?
+
+  // this holds the log level if not overridden
+  private var _logLevel: Logger.Level = .info
+
+  // metadata storage
+  public var metadata: Logger.Metadata = [:]
+
+  public init(label: String) {}
+
+  public var outputs = [Outputs.file]
+
+  public var _path: URL?
+
+  public var path: URL {
+    get {
+      guard let _path = _path else {
+        let fm = FileManager.default
+        guard let url = fm.urls(
+          for: FileManager.SearchPathDirectory.documentDirectory,
+          in: FileManager.SearchPathDomainMask.userDomainMask
+        )
+        .last?.appendingPathComponent("log.txt") else {
+          return URL(fileURLWithPath: "")
+        }
+        return url
+      }
+      return _path
+    }
+
+    set {
+      _path = newValue
+    }
+  }
+
+  public var logLevel: Logger.Level {
```

Comment:
`_logLevel` isn't used anywhere other than this computed variable. It would be much simpler if `logLevel` was a simple stored property without proxying to `_logLevel`.